src/overlays/actors/ovl_En_Ge3/z_en_ge3.c

```diff
@@ -237,10 +238,10 @@ s32 EnGe3_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* p
         // Turn head
         case GELDB_LIMB_HEAD:
             rot->x += this->headRot.y;
-#if PLATFORM_GC
+#if OOT_VERSION >= PAL_1_1
             FALLTHROUGH;
-        // This is a hack to fix the color-changing clothes this Gerudo has on N64 versions
         default:
+            // This is a hack to fix the color-changing clothes this Gerudo has on before below PAL 1.1
             OPEN_DISPS(play->state.gfxCtx, "../z_en_ge3.c", 547);
```

Comment:
"has on before below" is a bit confusing
maybe `// This is a hack to fix the bug present before PAL 1.1, where the actor's clothes can change color`

Reply:
"before below" was a typo, thanks. I took your suggestion and expanded a tiny bit